### PR TITLE
chore: setup the page layout , the mobile and the desktop view

### PR DESCRIPTION
**Pull request by Kareem24**
### Test Plan

1. I checked the page layout on desktop and mobile view 
2. Change the web address to route to each pages in the application 
3. To change page route
- edit the web address by adding "/win" to the end of the website address to change page to the win page

### CODE REVIEW

**Comment by BasitKorede** [comment]
image

##### This shouldn't be the design for the mobile screen.


See https://www.frontendmentor.io/challenges/rock-paper-scissors-game-pTgwgvgH for the correct design.

File: src/components/Mainpage.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+import scissors from '../assets/images/icon-scissors.svg'
+import paper from '../assets/images/icon-paper.svg'
+import rock from '../assets/images/icon-rock.svg'
+import triangle from '../assets/images/bg-triangle.svg'
 const Mainpage = () => {
   return (
-    <div>
-      <header className='flex justify-between py-1 px-3 border-white rounded w-[50%] border m-auto items-center'>
+    <>
+      <header className='flex justify-between py-1 px-3 border-headerOutline rounded md:w-[60%] w-[90%] border m-auto items-center'>
```
Comment (BasitKorede) [inline src/components/Mainpage.jsx]: I would prefer we use tailwind values for our CSS properties, all values given to the properties in most occurrences have their tailwind representation. 
eg `w-[60]` should be  `w-3/5`

File: tailwind.config.js [hunk 2]
```diff
@@ -15,7 +15,16 @@ export default {
         cyanStart: 'hsl(189, 59%, 53%)',
         cyanStop: ' hsl(189, 58%, 57%)',
         scissorsStart: 'hsl(39, 89%, 49%)',
+        scissorsEnd: 'hsl(40, 84%, 53%)',
+        rockEnd: 'hsl(349, 70%, 56%)',
+        rockStart: 'hsl(349, 71%, 52%)',
+        start: 'hsl(214, 47%, 23%)',
+        end: 'hsl(237, 49%, 15%)',
```
Comment (BasitKorede) [inline tailwind.config.js:22]: can you please explain the reason for this in the config file? I don't know what this is you can add your comment below this comment. Thank you

**Review by BasitKorede — CHANGES_REQUESTED** [review]
Good job done  Mr @Kareem24 , but unfortunately, there are still some changes to be made.
- The pull request description isn't descriptive enough.  Let's try to add all changes made in this particular branch.
- I noticed the test plan forum wasn't in use either.
let's make sure to add a test plan. this will entail how someone viewing the PR can access and test the project being done. Below is an example of what a test plan should look like.


image
Once again good job 👍

File: tailwind.config.js (hunk 2 above)
Comment (Kareem24) [reply tailwind.config.js:22]: I'm setting the color variable, so I can access them easily on my code

**Review by BasitKorede — APPROVED** [review]
Good job done 🥳

**Pull request merged**